transformer_engine/pytorch/module.py

```diff
@@ -457,7 +459,7 @@ def prepare_forward(
 
             # Previous iteration was grad_enabled
             if self.fp8_meta.get("update_amax_and_scale_fwd", False):
-                if self.fp8_meta["recipe"].reduce_amax:
+                if self.fp8_meta["recipe"].reduce_amax or self.sequence_parallel:
```

Comment:
I think there should be some sort of one-time warning if those two settings are set differently, since you are basically overriding the user here -> they should know that the amax reduction actually still happens.